src/containers/NoteItem/NoteItem.js

```diff
@@ -38,4 +39,15 @@ export class NoteItem extends Component {
   }
 }
 
+NoteItem.propTypes = {
+  handleItemChange: PropTypes.func,
+  handleItemDelete: PropTypes.func,
+  item: PropTypes.object,
+  toggleComplete: PropTypes.func,
+}
+
+NoteItem.defaultProps = {
```

Comment:
Thanks for adding these in! I also didn't know about .defaultProps - will make sure I incorporate that in projects moving forward!